src/components/MarketplaceItemDetailPage.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <div class="detail-page">
+    <div class="heading-container">
+      <div class="title">
+        <h2>{{ item.name }}</h2>
+      </div>
+      <div class="author">by {{ item.author }}</div>
+    </div>
+    <div class="content-container">
+      <div class="left-panel">
+        <div class="thumbnail-container">
+          <img v-bind:src="item.thumbnailUrl" />
+        </div>
+        <div class="categories-container">
+          <h3>Category</h3>
+          <p>{{ item.category }}</p>
+        </div>
+        <div class="actions-container">
+          <button
+            v-on:click="goToProject(item)"
+            v-bind:data-tracking-label="item.name"
+            v-bind:href="item.sourceUrl"
+            class="btn btn--cta item-action-button"
+          >Take me to the Project</button>
```

Comment:
Are we OK with the casing there?

Reply:
And the content itself?
Project -> Extension?

Reply:
we'll stay with project since it could be project, site, repository etc.